packages/components/src/components/hint-box/hint-box.jsx

```diff
@@ -6,11 +6,14 @@ import Icon from '../icon/icon.jsx';
 const HintBox = ({ className, icon, is_info, is_warn, message }) => {
     return (
         <div
-            className={classNames('dc-hint-box', {
-                'dc-hint-box--info': is_info,
-                'dc-hint-box--warn': is_warn,
-                className: !!className,
```

Comment:
`className` is to be used as a dynamic key. Hence the above syntax will not work. JS will create an object with property `className` and set the coerced boolean value